kmq.js

```diff
@@ -75,7 +75,30 @@ client.on("message", (message) => {
                 message.channel.send(`The new cutoff year is \`${gameSession.getBeginningCutoffYear()}\`.`);
             }
         }
+        else if (command.action === "gender") {
+            if (gameSession.setGender(command.components)) {
```

Comment:
You can just have `setGender` return the gender array instead of exposing `getGenderArray`

Reply:
That'll make handling invalid input a bit harder, as I won't be able to differentiate between `!gender female` and garbage input.

Reply:
You can return an empty array if input is garbage, then this `if` statement won't execute